src/Http/Controllers/StaticPagesUserController.php

```diff
@@ -8,21 +8,21 @@
 class StaticPagesUserController extends Controller
 {
 
-    public function pageList()
+    public function pageList(StaticPage $staticPages)
     {
-        if (view()->exists('packages.static-pages.pages-list')) {
-            return view('packages.static-pages.pages-list');
-        } else {
-            return view('staticPages::site.pages-list');
-        }
+        $pages = $staticPages->where('status', $staticPages::STATUS_ACTIVE)->orderBy('updated_at', 'desc')->get();
```

Comment:
`$staticPages::STATUS_ACTIVE` - this is wrong. Don't use instance for constants.
Call constants only in static way.
`StaticPages::STATUS_ACTIVE`